runtime/vm/ContinuationHelpers.cpp

```diff
@@ -88,19 +88,39 @@ createContinuation(J9VMThread *currentThread, j9object_t continuationObject)
 	return result;
 }
 
+void
+synchronizeWithConcurrentGCScan(J9VMThread *currentThread, J9VMContinuation *continuation)
+{
+	/* atomically 'or' (not 'set') continuation->carrierThread  with currentThread */
+	uintptr_t oldValue = VM_AtomicSupport::bitOr((volatile uintptr_t*)&continuation->carrierThread, (uintptr_t)currentThread);
+
+	if (oldValue&J9_GC_CONTINUATION_CONCURRENTSCANNING) {
+		/* (currentThread was low tagged (GC was already in progress) */
+		internalReleaseVMAccess(currentThread);
+
+		volatile uintptr_t *localAddr = (volatile uintptr_t *) &continuation->carrierThread;
+		omrthread_monitor_enter(currentThread->publicFlagsMutex);
+		while ((((uintptr_t)*localAddr) & J9_GC_CONTINUATION_CONCURRENTSCANNING)) {
```

Comment:
ANY_BITS here too. The case also seems unnecessary given the pointer type.

Reply:
you probably meant 'cast'